py-polars/tests/unit/meta/test_versions.py

```diff
@@ -12,4 +12,5 @@ def test_show_versions(capsys: Any) -> None:
     out, _ = capsys.readouterr()
     assert "Python" in out
     assert "Polars" in out
+    assert "LTS CPU" in out
```

Comment:
There are some other fields that we can test for - should i add those too?

Reply:
What were you thinking about testing? Either way, not in this PR.

Reply:
`Version info` `Index type` and `Platform`